SparkPost/2web2ui
# TR-2709 - Reflect the new Delivery Confidence Score into the App

[TR-2709](https://sparkpost.atlassian.net/browse/TR-2709)

### What Changed
- Adds new `delivery_confidence_score` to the `/recipient-validation/single/<email>` page
- Adds supporting tests
- Updates styles slightly to accommodate wider text
- After encountering a warning while running Jest, updated the `ExternalLink` component to no longer define a styled component within another React component

### How To Test
- Go to `/recipient-validation/single` and validate an email address on an account with a credit card
- Verify the delivery confidence score renders both in the list and the raw API response section
- Verify tests adequately capture acceptance criteria
- Verify instances of `ExternalLink` still render correctly

### To Do
